migration/display-maintenance-screen.sh

```diff
@@ -1,41 +0,0 @@
-#!/bin/bash
-
-ENV=$1
-STARTTIME=$2
-ENDTIME=$3
-
-NAMESPACE="hmpps-book-a-video-link-$ENV"
-SECRETS_FILE="book-a-video-link-secrets.yaml"
-
-# Create the secrets YAML file
-cat <<EOF > $SECRETS_FILE
-apiVersion: v1
-kind: Secret
-metadata:
-  name: maintenance
-  namespace: $NAMESPACE
-type: Opaque
-stringData:
-  maintenance_mode: "true"
-  maintenance_start_datetime: "$STARTTIME"
-  maintenance_end_datetime: "$ENDTIME"
-EOF
-
-# Apply the secret
-kubectl apply -f ./$SECRETS_FILE
-echo "Applied secrets to $NAMESPACE"
```

Comment:
Matt - can we keep this one please? Think we'll want the ability to put up the screen..